components/scrape_it_cloud/actions/start-scraping/start-scraping.mjs

```diff
@@ -0,0 +1,91 @@
+import app from "../../scrape_it_cloud.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "scrape_it_cloud-start-scraping",
+  name: "Start Scraping",
+  description: "Initiate the scraping process for a specific endpoint. [See the documentation here](https://scrape-it.cloud/docs/api-features/basic-request).",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    url: {
+      type: "string",
+      label: "URL",
+      description: "Specify the URL of the web page you would like to scrape.",
```

Comment:
```suggestion
  description: "Specify the URL of the web page you want to scrape."
```

Reply:
Changed "would like to" to "want to" as per Google's Developer docs style guide.